frontend/src/components/dashboard/MineList.js

```diff
@@ -1,83 +1,132 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 import { Link } from "react-router-dom";
-import { Col, Row, Divider } from "antd";
+import { Table } from "antd";
+import { ELLIPSE, RED_ELLIPSE } from "@/constants/assets";
 import * as router from "@/constants/routes";
+import * as String from "@/constants/strings";
 import NullScreen from "@/components/common/NullScreen";
 
+const columns = [
+  {
+    title: "Mine Name",
+    width: 200,
+    dataIndex: "mineName",
+    render: (text, record) => <Link to={router.MINE_SUMMARY.dynamicRoute(record.key)}>{text}</Link>,
+  },
+  {
+    title: "Mine #",
+    width: 100,
+    dataIndex: "mineNo",
+  },
+  {
+    title: "Operational Status",
+    dataIndex: "operationalStatus",
+    width: 150,
+    render: (text, record) => (
+      <div className="inline-flex">
+        <div>
+          {record.operationalValue && (
+            <img
+              className="ellipse--small"
+              src={record.operationalValue === "OP" ? ELLIPSE : RED_ELLIPSE}
+              alt="ellipse"
+            />
+          )}
+        </div>
+        <div>{text}</div>
+      </div>
+    ),
+  },
+  {
+    title: "Permit #",
+    dataIndex: "permit",
+    width: 150,
+    render: (text, record) => (
+      <div>
+        {text && text.map(({ permit_no, permit_guid }) => <div key={permit_guid}>{permit_no}</div>)}
```

Comment:
Does this add a linting camelCase error? We'll need to disable that rule if we're going to write our code this way anyway

Reply:
it does indeed add the linting error. 

I was thinking the same thing.